### ISSUE TITLE
NVDA doesn't read the logon dialog under Windows 8 after a person logs off which doesn't use NVDA

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4158]
**Reported by heikofolkerts on 2014-05-27 17:56**
For the following szenario we take two users called bill for blind user and Sarah for a sighted user.
When the System starts up the logon dialogue is read by NVDA. This also holds true after Bill has logged on, used NVDA and logged off. But after Sarah has logged on (not using NVDA) and logs off the logon dialogue is not read anymore by NvDA. Bill can only reboot the System or try to logon without screen reader support.

When bill (which in our situation is a user with a microsoft account while Sarah is only a local user) doesn't log off but changes the user (from within Ctrl+alt+Del and Sarah logs on and doesn't use NVDA and logs off the logon dialog is read by NVDA. So it seems to depend on the fact that Bill is still logged in in this situaltion.

**Blocked by #308**

**Comment by bhavyashah** [comment]
@jcsteh According to your https://github.com/nvaccess/nvda/issues/4158#issuecomment-155320791, you seem to believe that #308  may have resolved the reported issue. Has it done so?

**Comment by jcsteh** [comment]
No idea. Needs to be tested.

**Comment by leonardder** [comment]
I'm pretty sure that this now works as expected, and will therefore close this as works for me. If this doesn't work with EOA as accepted, please open a new issue following the issue template.
